### PR TITLE
Onboarding Vendor and Public Officer Flow

### PR DESCRIPTION
**Pull request by komalrpande**
Updated onboarding flow(User perspective) for Vendor and Public officer

### CODE REVIEW

File: docs/onboard.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Onboard
+
+## Vendor
+
+### Step 1
+Send in your request for a TechPass account to TechPass Team
```
Comment (yilunyou) [inline docs/onboard.md:6]: to the product team.  it's our tenant admins who will create these vendor accounts

File: docs/onboard.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+![Vendor_TermsOfPolicy](assets/support/Vendor_TermsOfPolicy.png)
+
+### Step 7
+TechPass portal will be opened and you will see My Profile section
+![Vendor_MyProfile](assets/support/Vendor_MyProfile.jpeg)
+
+## Public Officer
+
+### Step 1
+Send in your request for a TechPass account to TechPass Team
```
Comment (yilunyou) [inline docs/onboard.md:49]: public officers will do self sign up via gsib machine.
you can update this instruction in the next pr

File: docs/onboard.md (hunk 1 above)
Comment (JLeeChan) [inline docs/onboard.md]: For vendor accounts, they need to be invited by tenant admins so that needs to be stated here.

File: docs/onboard.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+# Onboard
+
+## Vendor
```
Comment (JLeeChan) [inline docs/onboard.md:3]: Do we have a page that describes the differences of Public Officer accounts and Vendor accounts?

Comment (yilunyou) [reply docs/onboard.md:3]: those are content damien and denise have plans to extract the content from private docs.

this set of changes by komal is purely on onboarding process

**Pull request merged**